### PR TITLE
476 Broken footer and mobile menu styles in IE10/9

### LINKED ISSUE #476: Broken styles IE9/IE10 Mobile Menu, Footer, Help Container
IE 10/ Win7 > http://prntscr.com/e31b4x

### PR DESCRIPTION
**Pull request by jamesjacobs**
ie9 fix
ie10 fix

Fixes #476

### CODE REVIEW

File: stylesheets/_component.footer.scss [hunk 1]
```diff
@@ -57,23 +59,6 @@
...
-}
-
-.lt-ie10.open-help .footer {
-    right: 300px;
-
-    @include respond-min($screen-desktop) {
-        right: 0;
-    }
-}
-
```
Comment (jamesjacobs) [inline stylesheets/_component.footer.scss]: These were unneeded

File: stylesheets/_layout.tabs.scss [hunk 2]
```diff
@@ -1,7 +1,3 @@
-.container {
-    height: 100%;
-}
-
```
Comment (jamesjacobs) [inline stylesheets/_layout.tabs.scss]: Duplicate container styles - same styles in `_structure.scaffolding.scss`

File: stylesheets/_layout.tabs.scss [hunk 3]
```diff
@@ -46,10 +42,6 @@
         left: auto;
         position: static;
     }
-
-    .lt-ie10 & {
-        left: 0;
-    }
```
Comment (jamesjacobs) [inline stylesheets/_layout.tabs.scss]: Unneeded

File: stylesheets/_layout.tabs.scss [hunk 4]
```diff
@@ -33,7 +29,7 @@
     -webkit-transition: -webkit-transform 500ms ease;
     backface-visibility: hidden;
     display: table;
-    height: 100%;
+    height: 100vh;
```
Comment (jamesjacobs) [inline stylesheets/_layout.tabs.scss]: vh's do work in IE9/10 - http://caniuse.com/#feat=viewport-units lied!

**Comment by jamesjacobs** [comment]
Tested in 

- [x]  IE9
- [x]  IE10
- [x]  IE11
- [x]  Edge
- [x]  Safari
- [x]  Firefox
- [x]  Chrome

**Comment by Stanton** [comment]
Seems ok in IE8 too

**Pull request merged**